src/lib/application/files/js/src/main.js

```diff
@@ -4,5 +4,9 @@ import { AppModule } from './app.module';
 async function bootstrap() {
   const app = await NestFactory.create(AppModule);
   await app.listen(3000);
+
+  const url = await app.getUrl();
+  // tslint:disable-next-line: no-console
```

Comment:
We can bypass this error message by using
`process.stdout.write(...)` - but I thought `console.log()` more well-known